DFC.Digital/DFC.Digital.Data/Model/JobProfileIndex.cs

```diff
@@ -57,5 +57,7 @@ public class JobProfileIndex : IDigitalDataModel
 
         [IsFilterable]
         public IEnumerable<string> JobAreas { get; set; }
+
+        public bool IsImported { get; set; }
```

Comment:
Don't think we need this, we dont need to update the index with imported content.

Reply:
We still need IsImported in JobProfileIndex for nameof. Otherwise  "IsImported" or constant.IsImported

Reply:
Now that you are using JobProfile.IsImported in nameof, this can be removed?